packages/form-js-viewer/src/render/components/form-fields/Taglist.js

```diff
@@ -0,0 +1,158 @@
+import { useContext, useEffect, useRef, useState } from 'preact/hooks';
+
+import { FormContext } from '../../context';
+
+import CloseIcon from './icons/Close.svg';
+
+import Description from '../Description';
+import Errors from '../Errors';
+import Label from '../Label';
+
+import {
+  formFieldClasses,
+  prefixId
+} from '../Util';
+import DropdownList from './parts/DropdownList';
+
+const type = 'taglist';
+
+
+export default function Taglist(props) {
+  const {
+    disabled,
+    errors = [],
+    field,
+    value : values = []
+  } = props;
+
+  const {
+    description,
+    id,
+    label,
+    values : options
+  } = field;
+
+  const { formId } = useContext(FormContext);
+  const [filter, setFilter] = useState('');
+  const [selectedValues, setSelectedValues] = useState([]);
+  const [filteredValues, setFilteredValues] = useState([]);
+  const [isDropdownExpanded, setIsDropdownExpanded] = useState(false);
+  const [hasValuesLeft, setHasValuesLeft] = useState(true);
+  const [escapeClose, setEscapeClose] = useState(false);
+  const searchbar = useRef();
```

Comment:
```suggestion
  const searchbarRef = useRef();
```

I think it is good practice, as we are not storing the DOM element, but the complete ref.